include/zephyr/posix/unistd.h

```diff
@@ -55,6 +55,153 @@ pid_t getpid(void);
 unsigned sleep(unsigned int seconds);
 int usleep(useconds_t useconds);
 
+#ifdef CONFIG_POSIX_SYSCONF
+#define SUPPORTED_POSIX1_VERSION 200809L
```

Comment:
This might be better if it were `_POSIX_VERSION`, and defined outside of `#ifdef CONFIG_POSIX_SYSCONF`.

The spec says that `_POSIX_VERSION` should be defined in `unistd.h` (newlib and picolibc define it in `sys/features.h`).

I think `#define _POSIX_VERSION 200809L` in this file should be OK though - maybe closer to the top?